app/main.js

```diff
@@ -33,23 +45,27 @@ const appEnter = (nextState, replace, callback) => {
       replace('/');
     callback();
   }));
-}
+};
+
 const newPlanEnter = (nextState, replace) => {
```

Comment:
I'm not sure which error you're seeing? I also didn't write this part so I assumed we wanted to keep it that way when I was resolving conflicts